deploy/cloudformation/layers/sentry/template.yml

```diff
@@ -0,0 +1,35 @@
+Transform: 'AWS::Serverless-2016-10-31'
+Description: >
+  - Sentry layer
+
+Resources:
+
+  SentryLayer:
+    Type: "AWS::Serverless::LayerVersion"
+    DeletionPolicy: Retain
+    UpdateReplacePolicy: Retain
+    Properties:
+      LayerName: "marble-sentry"
```

Comment:
Should prob still include stack name in this somehow. Otherwise you'll have collisions if you want to deploy dev/test/prod layers